data-tool/flows/batch_delete_flow.py

```diff
@@ -256,6 +352,83 @@ def colin_delete(config, db_engine: Engine, identifiers: list):
             delete_by_ids(conn, 'corp_processing', plan['corp_processing'])
 
 
+@task
+def auth_api_delete(config, identifiers: list):
```

Comment:
Similar situation as that in #3086, reuse the code in legal-api with some tweaks to get rid of the dependency on it.